### PR TITLE
chore: Update documentation about responsive utils

### PR DESCRIPTION
**Pull request by GoOz**
Update documentation so responsive suffixes for utility classes are officially explained and not only buried in the code.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
   "homepage": "https://github.com/cozy/cozy-ui",
   "scripts": {
     "build:css": "env CSSMODULES=false stylus -c --include stylus -o dist/cozy-ui.min.css stylus/cozy-ui/build.styl --use ./node_modules/autoprefixer-stylus --with \"{ browsers: ['last 2 versions'] }\"",
-    "build:css:app": "env CSSMODULES=false stylus --include stylus -o build/styleguide/app.css stylus/cozy-ui/build.styl --use ./node_modules/autoprefixer-stylus --with \"{ browsers: ['last 2 versions'] }\" --include-css",
+    "build:css:app": "env CSSMODULES=false stylus -c --include stylus -o build/styleguide/app.css stylus/cozy-ui/build.styl --use ./node_modules/autoprefixer-stylus --with \"{ browsers: ['last 2 versions'] }\"",
```
Comment (y-lohse) [inline package.json]: `-c` is the new `--include css`?

File: stylus/settings/breakpoints.styl [hunk 2]
```diff
@@ -22,21 +22,35 @@ breakpoints = {
...
+
+    This is the collection of available breakpoint variables:<br><br>
+    - `BP-teeny`         -  375px
+    - `BP-tiny`          -  480px
+    - `BP-small`         -  768px
+    - `BP-medium`        -  1023px
+    - `BP-large`         -  1200px
+    - `BP-extra-large`   -  1439px
+
+    <br>Also we have a standard breakpoints collection meant to be used as suffixes for some utilities<br>
```
Comment (y-lohse) [inline stylus/settings/breakpoints.styl]:
```suggestion
    <br>Also we have a standard breakpoints collection meant to be used as suffixes for some utility classes<br>
```

File: stylus/settings/breakpoints.styl [hunk 3]
```diff
@@ -22,21 +22,35 @@ breakpoints = {
...
+
+    If you want a utility class to trigger only at defined breakpoints you can use those:<br><br>
```
Comment (y-lohse) [inline stylus/settings/breakpoints.styl]:
```suggestion
    If you want a utility class to trigger only at a defined breakpoints you can use those:<br><br>
```

File: stylus/settings/breakpoints.styl [hunk 4]
```diff
@@ -22,21 +22,35 @@ breakpoints = {
...
+    - `-s`  - refers to `tiny` breakpoint (max-width: 480px)
```
Comment (y-lohse) [inline stylus/settings/breakpoints.styl]:
```suggestion
    - `-t`  - refers to `tiny` breakpoint (max-width: 480px)
```

File: package.json (hunk 1 above)
Comment (GoOz) [reply package.json]: Not really no. 
`--include-css` allows to `@import` css file which we did until #833 There's no need anymore.
`-c` is for CSS compression.

File: stylus/settings/breakpoints.styl (hunk 3 above)
Comment (GoOz) [reply stylus/settings/breakpoints.styl]: Are you sure? I wrote "breakpoints", plural. You shouldn't add "a", should you?

Comment (y-lohse) [reply stylus/settings/breakpoints.styl]: No you're completely right, sorry.

**Pull request merged**